examples/SimpleQA/SimpleQA.ipynb

```diff
@@ -0,0 +1,384 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Evaluating a Question-Answering system\n",
+    "\n",
+    "We're going to evaluate a simple QA system using [autoevals](https://github.com/braintrustdata/autoevals), our built-in library for evaluating AI model outputs. By the time you finish this example, you'll learn how to define and use custom evaluation metrics, and analyze results in Braintrust."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "\n",
+    "Before getting started, make sure you have a [Braintrust account](https://www.braintrust.dev/signup) and an API key for [OpenAI](https://platform.openai.com/). Make sure to plug the OpenAI key into your Braintrust account's [AI providers](https://www.braintrust.dev/app/settings?subroute=secrets) configuration and acquire a [BRAINTRUST_API_KEY](https://www.braintrust.dev/app/settings?subroute=api-keys). Add your `BRAINTRUST_API_KEY` to a `.env` file in your directory. \n",
+    "\n",
+    "### Install dependencies\n",
+    "\n",
+    "Everything you need to run evals is readily available through Braintrust. Although we'll use OpenAI models, you can use a wide variety of models through the [AI proxy](https://www.braintrust.dev/docs/guides/proxy) without having to write model-specific code. Run the following command to install required libraries."
```

Comment:
not sure the proxy statement adds anything here. I think when we use it below, maybe we mention it + the fact that it caches things for you?

Reply:
Also, now that i think about it, maybe we should update this cookbook to compare gpt-4o to LLaMa 3.3 or something? Maybe then the proxy is more obvious?